### PR TITLE
Unicorn bump npm packages

### PR DESCRIPTION
**Pull request by brev**
Numenta-apps Unicorn
* Bump npm packages
** Skips upgrades to:
*** Node: should be on 4.2.2, but not sure what to do on https://github.com/NumentaCorp/numenta-private-salt-infrastructure/pull/320
*** Babel: Babel 6 missing decorator support still
*** Electron-prebuilt: `ipc` deprecation
* fix lints
* remove old Web mentions
* etc.

### CODE REVIEW

**Comment by brev** [comment]
@lscheinkman @marionleborgne please review, thanks.

File: unicorn/js/browser/components/Chart.jsx [hunk 1]
```diff
@@ -162,6 +162,12 @@ export default class Chart extends React.Component {
 
   /**
    * DyGrpahs Chart range finder change/zoom callback function
+   * @method
+   * @param {Number} rangeXmin - Minimum X value of chart Range Finder
+   * @param {Number} rangeXmax - Maximum X value of chart Range Finder
+   * @param {Array} [yRanges] - Extra Y value data (unused currently)
+   * @private
+   * @this Chart
```
Comment (lscheinkman) [inline unicorn/js/browser/components/Chart.jsx]: There is no need for the extra `jsdoc` tags. We are using `esdoc` and it supports `ES6` classes. See https://esdoc.org/feature.html#es6-class  and https://esdoc.org/tags.html

File: unicorn/js/browser/components/FileList.jsx [hunk 2]
```diff
@@ -125,12 +136,12 @@ export default class FileList extends React.Component {
     }
   }
 
-  _handleMetricContextMenu(modelId, filename, timestampField, metric, ev, action) {
+  _handleMetricContextMenu(modelId, filename, tsField, metric, event, action) {
```
Comment (lscheinkman) [inline unicorn/js/browser/components/FileList.jsx]: Why rename `timestampField` to `tsField`?

File: unicorn/js/browser/components/FileList.jsx [hunk 3]
```diff
@@ -44,14 +52,17 @@ const {
 const DIALOG_STRINGS = {
   model: {
     title: 'Delete Model',
-    message: 'Deleting this model will delete the associated model results. Are you sure you want to delete this model?'
+    message: `Deleting this model will delete the associated model results.
+              Are you sure you want to delete this model?`
```
Comment (lscheinkman) [inline unicorn/js/browser/components/FileList.jsx]: This may add extra spaces to the string.

**Pull request closed without merging**